### ISSUE TITLE
On desktop, projects list is incomplete

### ISSUE THREAD

**Issue by matthias-ronge** [kitodo/kitodo-production#3388]
On the desktop page, the projects list is incomplete if there are few more projects than the limit allows. The general assumption here is that the number of projects existing on the system is always very limited (maybe 30 or so) so that all projects can always be displayed. On the other hand, creating a new process for a project (plus button) is extremely important and should be accessible for all projects (except ended ones) from here.

**Goal:** Make projects list on desktop unlimited.

**Comment by solth** [comment]
No, general assumption is not that the number of projects is so limited that they all fit on the desktop widget. Instead, the desktop widgets are merely meant as previews for what's in the system. They do not support paging through lists, the have a limited number of actions and they cannot be sorted or filtered by the user (otherwise they'd provide nearly all functionality of the individual list pages and thus would make those obsolete.
AFAIK this is a design decision and not an oversight.

**Comment by matthias-ronge** [comment]
Arved, what you're saying is true (at the moment). Therefore, this ticket is not marked as a bug, it is rather a kind of change request. When testing with live data, it was noticed that this is very annoying, and the assumption that there are not masses of projects on a system can be made. What do you think, maybe we should just make the limit configurable?
